modules/textures/src/lib/parsers/parse-basis.ts

```diff
@@ -204,7 +204,6 @@ function parseKTX2File(KTX2File, data: ArrayBuffer, options): TextureLevel[][] {
 
     for (let levelIndex = 0; levelIndex < levelsCount; levelIndex++) {
       levels.push(transcodeKTX2Image(ktx2File, levelIndex, options));
-      break; // texture app can only show one level for some reason
```

Comment:
@belom88 or @ibgreen do you know what "texture app" refers to here? Is that something I would need to test before merging the PR?

Reply:
I assume this would be https://loaders.gl/examples/textures